store/storage/rocksdb/db.go

```diff
@@ -120,27 +100,22 @@ func (db *Database) GetLatestVersion() (uint64, error) {
 }
 
 func (db *Database) Has(storeKey string, version uint64, key []byte) (bool, error) {
-	if version < db.tsLow {
-		return false, nil
-	}
-
 	slice, err := db.getSlice(storeKey, version, key)
-	if err != nil {
+	if err != nil || slice == nil {
 		return false, err
 	}
 
 	return slice.Exists(), nil
 }
 
 func (db *Database) Get(storeKey string, version uint64, key []byte) ([]byte, error) {
-	if version < db.tsLow {
-		return nil, nil
-	}
-
 	slice, err := db.getSlice(storeKey, version, key)
 	if err != nil {
 		return nil, fmt.Errorf("failed to get RocksDB slice: %w", err)
 	}
+	if slice == nil {
+		return nil, nil
```

Comment:
could we return an error saying what the lowest height of the node is? this would be good information for those using rpcs